precompiles/staking/approve_test.go

```diff
@@ -155,7 +159,7 @@ func (s *PrecompileTestSuite) TestApprove() {
 			func(data []byte, inputArgs []interface{}) {
 				s.Require().Equal(data, cmn.TrueValue)
 
-				allAuthz, err := s.app.AuthzKeeper.GetAuthorizations(s.ctx, s.address.Bytes(), s.address.Bytes())
+				allAuthz, err := s.network.App.AuthzKeeper.GetAuthorizations(ctx, s.keyring.GetAddr(0).Bytes(), s.keyring.GetAddr(0).Bytes())
```

Comment:
unnecessary conversions


```suggestion
				allAuthz, err := s.network.App.AuthzKeeper.GetAuthorizations(ctx, s.keyring.GetAccAddr(0), s.keyring.GetAccAddr(0))
```